src/c-api.adoc

```diff
@@ -889,11 +889,16 @@ Each queryable extension must have an associated `groupid` and `bitmask` that in
 |====
 | extension | groupid | bit position
 | a | 0 | 0
+| b | 0 | 1
 | c | 0 | 2
 | d | 0 | 3
+| e | 0 | 4
 | f | 0 | 5
+| h | 0 | 7
 | i | 0 | 8
 | m | 0 | 12
+| s | 0 | 18
```

Comment:
s is a prefix isn't it?

Reply:
it is, reviewing the misa meaning, there's a slight difference between "implies extension" (most of them) vs "mode implemented" (U and S). I'll remove U and S.

there is a "X" misa bit with a meaning to mean "you have an extension starting X", which i didn't feel the need to add.